<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat</class>
 <widget class="QDialog" name="chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>421</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>423</width>
    <height>421</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>421</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="send_chat">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_chat">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>391</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <item>
     <widget class="QFontComboBox" name="fontComboBox"/>
    </item>
    <item>
     <widget class="QComboBox" name="front_size"/>
    </item>
    <item>
     <widget class="QToolButton" name="QQface">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>face/2.gif</normaloff>face/2.gif</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="front_color">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <activeon>face/textcolor.png</activeon>
       </iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="filetrans">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <activeon>face/fileopen.png</activeon>
       </iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_4">
      <property name="text">
       <string>记录</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>fontComboBox</zorder>
   <zorder>front_size</zorder>
   <zorder>QQface</zorder>
   <zorder>filetrans</zorder>
   <zorder>toolButton_4</zorder>
   <zorder>front_color</zorder>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>295</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>181</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
